package.json

```diff
@@ -94,6 +94,7 @@
     "lottie-react-native": "^3.3.2",
     "match-sorter": "^4.0.2",
     "nanoid": "^2.1.6",
+    "node-emoji": "omnidan/node-emoji#v2",
```

Comment:
@brunobar79 i dont think it will! 

basically we were using a package previously called `react-native-emoji` and it was using an old version of this library

i decided that using `react-native-emoji` was not worth it, and we now implement the functionality ourselves by using this `node-emoji` lib inside our `text/Emoji.js` component